revocation/ocsp/ocsp.go

```diff
@@ -0,0 +1,180 @@
+// Package OCSP provides methods for checking the OCSP revocation status of a certificate chain
+// as well as errors related to these checks
+package ocsp
```

Comment:
we should put this package under a folder with name `internal`.

Reply:
There are a few issues with this:
 - https://github.com/notaryproject/notation-core-go/pull/134#discussion_r1153303540
 - https://github.com/notaryproject/notation-core-go/pull/134#discussion_r1153316808